distributionviewer/core/static/js/app/components/views/chart-list.js

```diff
@@ -9,8 +9,8 @@ export default function(props) {
   }
 
   return (
-    <article className="chart-list">
+    <section className="chart-list">
```

Comment:
Hmm. I might have to agree to disagree. MDN says that `aside` "represents a section of the page with content connected tangentially to the rest, which could be considered separate from that content."

It seems like the charts are pretty central to the page, and even the sidebar is only visually on the side. But I could be convinced otherwise. :smile: